packages/flutter/lib/src/material/mergeable_material.dart

```diff
@@ -122,12 +122,8 @@ class MergeableMaterial extends StatefulWidget {
 
   /// The z-coordinate at which to place all the [Material] slices.
   ///
-  /// The following elevations have defined shadows: 1, 2, 3, 4, 6, 8, 9, 12, 16, 24.
-  ///
   /// Defaults to 2, the appropriate elevation for cards.
-  ///
-  /// This uses [kElevationToShadow] to simulate shadows, it does not use
-  /// [Material]'s arbitrary elevation feature.
+  // TODO(ianh): Change this to double.
```

Comment:
From a quick search in google3 it looks like it wouldn't actually break anything to just make that change?